src/js/decorators/prevent-clickjack.js

```diff
@@ -1,8 +1,17 @@
 var fireGAEvent = require( 'helpers/fire-ga-event' );
+var trustedUrls = [ 'https://fxa-support-panel.stage.mozaws.net/' ];
 
 module.exports = function() {
   if ( window.self !== window.top ) {
+    fireGAEvent( 'Security', 'NLX was loaded inside iframe: ' + window.self );
```

Comment:
I guess that log is intentional and it won't spam to much?

Reply:
yes - it was already tehre, just more detailled